envoy-control-tests/src/main/kotlin/pl/allegro/tech/servicemesh/envoycontrol/config/envoy/EnvoyContainer.kt

```diff
@@ -39,7 +39,7 @@ class EnvoyContainer(
         const val INGRESS_LISTENER_CONTAINER_PORT = 5001
         private const val ADMIN_PORT = 10000
 
-        private const val MIN_SUPPORTED_ENVOY_VERSION = "v1.22.7"
+        private const val MIN_SUPPORTED_ENVOY_VERSION = "v1.30.4"
```

Comment:
There is no previous envoy version that can pass tests.
`v1.30.4` fails on OutlierDetectionTest after [this PR](https://github.com/allegro/envoy-control/pull/449/files#diff-41a2e3a91f36ca63a79bfd5d2814273caf94508da4dde648bef5811b21aa6740)
So I leave it to you. 
Routing tests are passing on all versions from `v1.24.6` to `v1.23.0`